autoware_adapi_v1_msgs/perception/msg/PredictedObjectKinematics.msg

```diff
@@ -0,0 +1,5 @@
+geometry_msgs/PoseWithCovariance initial_pose_with_covariance
+geometry_msgs/TwistWithCovariance initial_twist_with_covariance
+geometry_msgs/AccelWithCovariance initial_acceleration_with_covariance
```

Comment:
```suggestion
geometry_msgs/PoseWithCovariance pose
geometry_msgs/TwistWithCovariance twist
geometry_msgs/AccelWithCovariance accel
```

Reply:
for this I think we can shorten to `pose_with_covariance`, to differentiate with pose.
How do you think?

Reply:
In [geometry_msgs](https://github.com/ros2/common_interfaces/blob/rolling/geometry_msgs/msg/PoseWithCovarianceStamped.msg), PoseStamped and PoseWithCovariance are also simply use `pose`, so I don't think there is much need to distinguish them.

Reply:
I see, okay, I will change it.

Reply:
changed at 
f6bfa5e